jenkins-scripts/docker/lib/_ros_setup_buildsh.bash

```diff
@@ -23,8 +23,8 @@ if ${USE_CATKIN_MAKE}; then
 else
   export CMD_CATKIN_CONFIG=""
   export CMD_CATKIN_LIST="colcon list -t"
-  export CMD_CATKIN_BUILD="colcon build --parallel-workers ${MAKE_JOBS} --symlink-install --event-handler console_direct+ ${CATKIN_EXTRA_ARGS}"
-  export CMD_CATKIN_TEST="colcon test --parallel-workers 1 --event-handler console_direct+ || true"
+  export CMD_CATKIN_BUILD="colcon build --parallel-workers ${MAKE_JOBS} --symlink-install --event-handler console_cohesion+ ${CATKIN_EXTRA_ARGS}"
+  export CMD_CATKIN_TEST="colcon test --parallel-workers 1 --event-handler console_cohesion+ || true"
```

Comment:
In this case where there are no concurrent workers keeping `console_direct+` makes sense to stream the output as it happens.

Same for the `test` invocation in Windows below.

Instead of `--parallel-workers` you could use `--executor sequential` - as it is used below already.

Reply:
thanks, should be fixed in b44c2b39cd8a6b9a5cc8263c68d11f534314a406